templates/flowfuse.yaml

```diff
@@ -0,0 +1,32 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+imports:
+  - grycap_custom_types: https://raw.githubusercontent.com/grycap/tosca/main/custom_types.yaml
+
+metadata:
+  template_version: "1.0.0"
+  template_name: FlowFuse
+  display_name: Deploy FlowFuse on a VM
+  icon: images/flowfuse.png
+  parents:
+    - simple-node-disk.yml
+
+description: Deploy FlowFuse service on a VM.
+
+topology_template:
+  node_templates:
+    flowfuse:
+      type: tosca.nodes.SoftwareComponent
+      artifacts:
+        docker_role:
+          file: grycap.flowfuse
+          type: tosca.artifacts.AnsibleGalaxy.role
+      properties:
+        port: 3000
+        protocol: tcp
```

Comment:
Check this:

      capabilities:
        endpoint:
          properties:
            port: 3000
            protocol: tcp